Parsers/ASimNetworkSession/Tests/PaloAlto_CortexDataLake_ASimNetworkSession_DataTest.csv

```diff
@@ -0,0 +1,7 @@
+Result
+"(0) Error: 2 invalid value(s) (up to 10 listed) in 6 records (200.0%) for field [EventVendor] of type [Enumerated]: [""Palo Alto Networks""] (Schema:NetworkSession)"
```

Comment:
@niralishah-crest 
duplicate entry. Palo Alto is already present. Please remove this change

Reply:
fixed in parser